src/pages/home/index.js

```diff
@@ -95,7 +96,12 @@ const Home = () => {
   } = useForm({ resolver: zodResolver(homeSchema) });
 
   const onSubmit = data => {
-    createTarget({ ...data, ...latLng });
+    if (targetsList.length <= 9) {
+      createTarget({ ...data, ...latLng });
+    } else {
+      setTargetsLimit(true);
```

Comment:
Are you doing anything with this `TargetsLimit`? Or you'll use it in a future feature?